ts_src/psbt.ts

```diff
@@ -659,6 +660,64 @@ export class Psbt {
     blindingPrivkeys: Buffer[],
     blindingPubkeys: Buffer[],
     opts?: RngOpts,
+  ): this {
+    return this.rawBlindOutputs(
+      blindingPrivkeys,
+      blindingPubkeys,
+      undefined,
+      opts,
+    );
+  }
+
+  blindOutputsByIndex(
+    inputsBlindingPrivKeys: Map<number, Buffer>,
+    outputsBlindingPubKeys: Map<number, Buffer>,
+    opts?: RngOpts,
+  ): this {
+    const blindingPrivKeysArgs = range(this.__CACHE.__TX.ins.length).map(
+      (inputIndex: number) => inputsBlindingPrivKeys.get(inputIndex),
+    );
+    const outputIndexes: number[] = [];
+    const blindingPublicKey: Buffer[] = [];
+
+    for (const [outputIndex, pubBlindingKey] of outputsBlindingPubKeys) {
+      outputIndexes.push(outputIndex);
+      blindingPublicKey.push(pubBlindingKey);
+    }
+
+    return this.rawBlindOutputs(
+      blindingPrivKeysArgs,
+      blindingPublicKey,
+      outputIndexes,
+      opts,
+    );
+  }
+
+  addUnknownKeyValToGlobal(keyVal: KeyValue): this {
+    this.data.addUnknownKeyValToGlobal(keyVal);
+    return this;
+  }
+
+  addUnknownKeyValToInput(inputIndex: number, keyVal: KeyValue): this {
+    this.data.addUnknownKeyValToInput(inputIndex, keyVal);
+    return this;
+  }
+
+  addUnknownKeyValToOutput(outputIndex: number, keyVal: KeyValue): this {
+    this.data.addUnknownKeyValToOutput(outputIndex, keyVal);
+    return this;
+  }
+
+  clearFinalizedInput(inputIndex: number): this {
+    this.data.clearFinalizedInput(inputIndex);
+    return this;
+  }
```

Comment:
can you put these methods where they were? Let's try to keep the existing other.
It's quite hard to review the core changes of the PR since it's plenty of unnecessary changes. Is it possible to polish the code?